code/game/gamemodes/colonialmarines/halloween2016.dm

```diff
@@ -773,7 +773,7 @@
 					if(12 to 15) //SADAR. the most popular choice, but also pretty damn deadly.
 						H.equip_to_slot_or_del(new /obj/item/clothing/head/helmet/marine(H), WEAR_HEAD)
 						H.equip_to_slot_or_del(new /obj/item/clothing/suit/storage/marine(H), WEAR_JACKET)
-						H.equip_to_slot_or_del(new /obj/item/storage/backpack/marine(H), WEAR_BACK)
+						H.equip_to_slot_or_del(new /obj/item/storage/backpack/marine/standard(H), WEAR_BACK)
```

Comment:
This is a both unneeded and undocumented change.

Reply:
If it's not done, they'll be stuck with the shitty base backpack which I'm essentially replacing.

Reply:
The satchel is not shitty, and it's the standard one.

Reply:
No, the shitty base _backpack_ which is the _root_ of the other backpacks/satchels.

Reply:
Alright.